src/api/observables.ts

```diff
@@ -115,6 +115,19 @@ export const bancorConverterRegistry$ = contractAddresses$.pipe(
   share()
 );
 
+export const stakingRewards$ = contractAddresses$.pipe(
+  pluck("StakingRewards"),
+  distinctUntilChanged(compareString),
+  shareReplay(1)
+);
+
+export const poolPrograms$ = stakingRewards$.pipe(
+  switchMap(() =>
+    vxm.rewards.fetchPoolPrograms()
```

Comment:
By calling ```fetchPoolPrograms``` without the param of ```stakingRewards``` are we assuming that it's going to be available through the getter in it's own method? Should we just accept a param of staking rewards here? Even optional if you wanted, that way we know it's going to be available and no side effects are necessary for this thing to run.

Reply:
sure i can add that